docs/advanced-node-scheduling.asciidoc

```diff
@@ -118,6 +119,7 @@ spec:
                 topologyKey: kubernetes.io/hostname
 ----
 
+[float]
 ==== Local Persistent Volume constraints
```

Comment:
I was thinking that for this document separate pages would actually make sense because if covers fairly independent topics. Would it in this case not make sense to just adjust the heading level

Reply:
I agree with you, multiple pages would make sense here 👍 

If that's OK with you I'd like to merge this first (docs repo master build looks broken because of it :|). Then check with @alaudazzi what we can do about separate pages.

Reply:
Fine with me.